<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-H10B0F2" failures="1" tests="3" name="Test" time="37.613" errors="0" timestamp="2022-10-10T15:25:14 IST">
  <testcase classname="com.sauceDemo.TestClasses.TC03_VerifyAddToCartFunctionality" name="verifyAddToCart" time="0.293">
    <failure type="java.lang.AssertionError" message="expected [5] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [5] but found [1]
at com.sauceDemo.TestClasses.TC03_VerifyAddToCartFunctionality.verifyAddToCart(TC03_VerifyAddToCartFunctionality.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyAddToCart -->
  <testcase classname="com.sauceDemo.TestClasses.TC02_VerifyLogoutFunctionality" name="verifyLogout" time="0.539"/>
  <testcase classname="com.sauceDemo.TestClasses.TC01_VerifyLoginFunctionality" name="loginFunctionality" time="0.005"/>
</testsuite> <!-- Test -->
